### ISSUE TITLE
Feature Request ♦ Shortcut: Enter to Open a Saved Page URL

### ISSUE THREAD

**Issue by touwys** [vctfence/scrapbee#59]
To open a saved page URL: hit keyboard "enter" or, mouse double-click, when the articel title is highlighted.

**Comment by vctfence** [comment]
Can be done, but I think shortcut like "ctrl + u"  or "ctrl + o" is better.  Using enter or double-click  here maybe cause confusion for many users.

**Comment by touwys** [comment]
Please explain how the "enter" key will cause confusion among users? Do you mean confusion to distinguish between the opening of the local file and the URL when the enter key is hit? However, this is not a big issue, and any shortcut key is to be welcomed — "ctrl + o" is fine. I have also noticed that the keyboard arrow keys (up and down) cannot be used inside the Sidebar to move around the tree. Is that by design, and can this behaviour be added?

_To take care of questions about keyboard-shortcuts: May I propose that instead of providing pre-chosen keyboard shortcuts as they arise, rather provide a configuration-element where the user may assign keyboard-shortcuts of his own?_

**Comment by vctfence** [comment]
Right, as I know "enter" or "double click" means "open content", for scrap nodes, it's local files, or remote page for bookmarks.  Though "it's not a big issue", it's enough to choose another key binding.

Key shortcuts are already can be set in Firefox's add-ons page, I'll research to see how it will work for this function.
